nyc_buildings/anaconda-project.yml

```diff
@@ -15,20 +15,26 @@ user_fields: [examples_config]
 
 channels:
 - defaults
```

Comment:
@Azaya89 ah I just noticed something. It's good practice not to mix the `defaults` channel with `conda-forge`. So when we use `conda-forge` we should replace `defaults` with `nodefaults`, to avoid the `defaults` channel to be added *by default* 🙃 Can you try that on your machine and re-lock?

Reply:
OK, but there was no failure in the CI here. This is diabolical!

Reply:
Done!